app/vmctl/opentsdb.go

```diff
@@ -82,6 +82,11 @@ func (op *otsdbProcessor) run(silent, verbose bool) error {
 		errCh := make(chan error)
 		// we're going to make serieslist * queryRanges queries, so we should represent that in the progress bar
 		bar := pb.StartNew(len(serieslist) * queryRanges)
+		defer func(bar *pb.ProgressBar) {
+			if !bar.IsFinished() {
+				bar.Finish()
+			}
```

Comment:
As for me it is strange behavior we stop bar if it is not finished. Maybe just call `bar.Finish()`?

Reply:
Sounds reasonable, implemented.